microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -432,7 +432,21 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
 
 Microsoft Edge offers an interactive and seamless experience when you launch the browser for the first time. You can import your browser data to Microsoft Edge from another browser. The data includes your favorites, browsing history, cookies, autofill data, extensions, settings, and other browsing data.
 
-With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.  Importing your data is completed locally on your device, and is stored locally, and is not sent to Microsoft unless you sign in and sync your browsing data.
+With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.
+
+Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 
+
+* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.
+
+* Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.
+<!-- todo: add directions for how to opt-out of personalization on Bing -->
+
+To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).
+<!-- todo: 
```

Comment:
@amatharoo1459 - 
* Provide a more direct link to view/delete the imported data.
* Include step-by-step instructions to delete the data.

Do those b/c the existing link goes to myaccount.microsoft.com and requires re-sign in to my account.  At that page, it still is not obvious how to view/delete the imported data.

per Point #3 in [PR Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files/b1c6d7a11cca3887a87f9790abe864231fe1d6ca#r1410027587)